src/pages/AdminDashboard/CreateRoomForm/createRoomTools.ts

```diff
@@ -46,8 +47,13 @@ export const validatePhoneNumber = (phoneNumber) => {
     return isValid
 }
 
-export const isParticipantValid = (participant) => {
-    return validateEmail(participant) || validatePhoneNumber(participant)
+export const validateGroupName = (groupName : string, groups : Array<Group>): boolean => {
+    const groupFound = groups.find(group => group.name === groupName);
+    return groupFound !== undefined;
```

Comment:
what about

```js
return group || false
```

Reply:
According to find documentation :

The value of the first element in the array that satisfies the provided testing function. Otherwise, undefined is returned.

Reply:
Yes, but your validate function returns a `boolean`. I just found this way of expressing that more elegant than testing against `undefined` value. I think the result should be the same :)